### DISCUSSION TITLE
Can I hide or remove the 'New' button from the table resource?

### DISCUSSION THREAD

**Discussion by cleargoal** [filamentphp/filament#12537] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

Can I hide or remove the 'New' button from the table resource?

I tried to find this info in the docs but I couldn't.

Thanks for any suggestions!

**Comment by leandrocfe** [comment, marked as answer]
Remove the CreateAction in the ListPage  file.

**Reply by cleargoal** [reply]
**Thank you very much!**
